### DISCUSSION TITLE
Niko 552-721X1 firmware upgrade

### DISCUSSION THREAD

**Discussion by Christophe0501** [Koenkk/zigbee2mqtt#25247] [Devices]
Hello,

I was wondering if it's possible to upgrade Niko devices (such as 552-721X1) without the Niko Home Control.

thanks in advance,

**Comment by Christophe0501** [comment]
just to clarify, I mean upgrade the firmware of the 552-721X1

**Comment by vdkeybus** [comment]
I know the question was posted a long time ago. As of version 2.6.2, it seems to be possible, yes.
